From a1180af9ee17684cbf5e28b443aea6d92c2688cf Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Tue, 10 Jan 2023 16:44:29 +0200
Subject: [PATCH] [SW-107923] gaudi3: export arcboot_rom_get_arc_addr api

This patch export's arcboot_rom_get_arc_addr arcboot ROM api so
that FW components can make use this api.

Change-Id: I63ce02ca7368d1d0df8058481b8a9f4bcc9f281f
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/arch/gaudi3/arc/arcboot_rom.txt b/arch/gaudi3/arc/arcboot_rom.txt
index 79cd821..5026001 100644
--- a/arch/gaudi3/arc/arcboot_rom.txt
+++ b/arch/gaudi3/arc/arcboot_rom.txt
@@ -24,3 +24,4 @@
 arcboot_rom_get_spi_manifest_hdr
 arcboot_rom_get_sec_data
 security_rr_open_pcie_config
+arcboot_rom_get_arc_addr
